packages/apps/plugins/plugin-ipfs/src/components/FilePreview.tsx

```diff
@@ -4,30 +4,21 @@
 
 import React, { type FC } from 'react';
 
+import { mx } from '@dxos/aurora-theme';
+
 export type FilePreviewProps = {
   type: string;
   url: string;
+  className?: string;
 };
 
 /**
  * File/content preview iframe.
  */
-export const FilePreview: FC<FilePreviewProps> = ({ type, url }) => {
+export const FilePreview: FC<FilePreviewProps> = ({ type, url, className }) => {
   if (!type.startsWith('image')) {
-    return <iframe className='w-full h-full overflow-auto' src={url} />;
+    return <iframe className={mx('w-full h-full overflow-auto', className)} src={url} />;
```

Comment:
So apparently only image types can currently be rendered from IPFS. But we could extend this area to support other types of things stored in IPFS.